docs/sources/datasources/azure-monitor/_index.md

```diff
@@ -141,7 +159,46 @@ For details on Azure managed identities, refer to the [Azure documentation](http
 
    This hides the directory ID, application ID, and client secret fields, and the data source uses managed identity to authenticate to Azure Monitor Metrics and Logs, and Azure Resource Graph.
 
-   {{< figure src="/media/docs/grafana/data-sources/screenshot-managed-identity.png" max-width="800px" class="docs-image--no-shadow" caption="Azure Monitor Metrics screenshot showing Dimensions" >}}
+   {{< figure src="/media/docs/grafana/data-sources/screenshot-managed-identity-2.png" max-width="800px" class="docs-image--no-shadow" caption="Azure Monitor Metrics screenshot showing Dimensions" >}}
+
+3. The `managed_identity_client_id` field can be set in the `[azure]` section of the [Grafana server configuration][configure-grafana-azure] to allow a user-assigned managed identity to be  used instead of the default system-assigned identity.
+
+  ```ini
+  [azure]
+  managed_identity_enabled = true
+  managed_identity_client_id = USER_ASSIGNED_IDENTITY_CLIENT_ID
+  ```
+   
+### Configure Workload Identity
+
+If you host Grafana in a Kubernetes environment, such as AKS, in conjunction with managed identities you can use workload identity to configure Azure Monitor in Grafana.
+This lets you securely authenticate data sources without manually configuring credentials via Azure AD App Registrations.
+For details on workload identity, refer to the [documentation](https://azure.github.io/azure-workload-identity/docs/).
+
+**To enable workload identity for Grafana:**
+
+1. Set the `workload_identity_enabled` flag in the `[azure]` section of the [Grafana server configuration][configure-grafana-azure].
+
+   ```ini
+   [azure]
+   workload_identity_enabled = true
+   ```
+
+2. In the Azure Monitor data source configuration, set **Authentication** to **Workload Identity**.
+
+   This hides the directory ID, application ID, and client secret fields, and the data source uses managed identity to authenticate to Azure Monitor Metrics and Logs, and Azure Resource Graph.
```

Comment:
```suggestion
   This hides the:
   - Directory ID
   - Application ID, and client secret fields
   - Data source uses managed identity to authenticate to Azure Monitor Metrics and Logs
   - Azure Resource Graph
```

Reply:
I don't think this change makes sense as the latter two items in the list shouldn't be there, is it okay to leave the original sentence structure?

Reply:
Yep. And I don't have authorization to merge in this repo, I don't think, so you will need to.